src/Sylius/Bundle/CoreBundle/Form/Type/ImageType.php

```diff
@@ -14,7 +14,7 @@
 use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
 use Symfony\Component\Form\FormBuilderInterface;
 
-class ImageType extends AbstractResourceType
+abstract class ImageType extends AbstractResourceType
```

Comment:
I've created new class `ProductVariantImageType` to avoid `TaxonImageType` extending `ImageType` (being the product variant one).